rdflib/compare.py

```diff
@@ -290,7 +292,7 @@ def _initial_color(self):
 
     def _individuate(self, color, individual):
         new_color = list(color.color)
-        new_color.append((len(color.nodes)))
+        new_color.append(tuple([len(color.nodes)]))
```

Comment:
sorry for not spotting this one before :-/

could also be written as `new_color.append((len(color.nodes),))`, which is a bit faster, but maybe premature optimization:

```python
In [1]: %timeit tuple([1337])
The slowest run took 5.53 times longer than the fastest. This could mean that an intermediate result is being cached
1000000 loops, best of 3: 345 ns per loop

In [2]: %timeit (1337,)
The slowest run took 47.08 times longer than the fastest. This could mean that an intermediate result is being cached
10000000 loops, best of 3: 25.3 ns per loop
```